code/hsec-sync/hsec-sync.cabal

```diff
@@ -0,0 +1,95 @@
+cabal-version:   2.4
+name:            hsec-sync
+version:         0.1.0.0
+
+-- A short (one-line) description of the package.
+synopsis:        Synchronize with the Haskell security advisory database
+
+-- A longer description of the package.
+description:     Synchronize with the Haskell security advisory database.
+
+-- A URL where users can report bugs.
+-- bug-reports:
+
+-- The license under which the package is released.
+license:         BSD-3-Clause
+author:          Gautier DI FOLCO
+maintainer:      gautier.difolco@gmail.com
+
+-- A copyright notice.
+-- copyright:
+category:        Data
+extra-doc-files: CHANGELOG.md
+tested-with:
+  GHC ==8.10.7 || ==9.0.2 || ==9.2.8 || ==9.4.8 || ==9.6.3 || ==9.8.1
```

Comment:
Not sure if it's wise to support so many GHC versions for a CLI tool. I would advise that you stay on the highest supported GHC at all times. Most people will anyway take your release binaries, especially in CI.

Reply:
Actually, CLI is more a bonus, I had flora/hackage-server in mind when I have made it.